FV3/atmos_cubed_sphere/model/fv_dynamics.F90

```diff
@@ -1560,5 +1602,13 @@ subroutine compute_aam(npz, is, ie, js, je, isd, ied, jsd, jed, gridstruct, bd,
   enddo
 
  end subroutine compute_aam
+ 
+ subroutine compute_column_integral(integrand, delp, column_integral)
+    real, intent(in), dimension(:,:,:) :: integrand, delp
+    real, intent(out) :: column_integral(:,:)
+    
+  column_integral = sum(integrand * delp, 3) / grav
+  
+ end subroutine compute_column_integral
```

Comment:
Would it be possible to reuse the implementation from here through a `use` statement at the top of the module, or was there a conflict?
https://github.com/ai2cm/fv3gfs-fortran/blob/291fb249ca59892f7e259fc3c5b7fbb2f02e2cd7/FV3/atmos_cubed_sphere/model/fv_update_phys.F90#L1223-L1229

Reply:
Good idea, I'll see if that works